core/app/models/spree/order.rb

```diff
@@ -171,9 +171,7 @@ def line_item_comparison_hooks=(value)
       line_item_hooks_deprecation_msg = "Use Spree::Config.line_item_comparison_hooks instead."
       deprecate :line_item_comparison_hooks= => line_item_hooks_deprecation_msg, :deprecator => Spree.deprecator
 
-      def line_item_comparison_hooks
-        Spree::Config.line_item_comparison_hooks
-      end
+      delegate :line_item_comparison_hooks, to: :'Spree::Config'
```

Comment:
I am not sure I am fan of this. We should probably adjust the Rubocop rules or use standard instead.